dsp/0002-state-of-the-metaverse.md

```diff
@@ -0,0 +1,312 @@
+
+### Other smart contracts
+
+The project relies on several other contracts deployed on the Ethereum network:
+
+- [**Burner contract**](https://github.com/decentraland/aux-contracts/tree/master/marketplace-burner): A contract that can only burn the ERC20 tokens it receives.
+- [**Auction**](https://github.com/decentraland/land-auction/tree/master/contracts/auction): A contract that was used for Decentraland's second LAND auction.
+- [**Passthrough**](https://github.com/decentraland/pass-through): A contract that can blacklist certain methods from being called to a target contract.
+- **Vesting**: Vesting contracts used to release tokens continuously over time.
+
+## Distribution and Verification of Content
+
+What is going on in the LAND parcel 13,37?
+
+In order to render the content and communicate with other users positioned on top of that parcel, the user agent needs to link those coordinates with a parcel definition (or parcel manifest). There are two ways to do this:
+
+1) The NFT has a URL and [CID](https://docs.ipfs.io/guides/concepts/cid/) that allows the user agent to use the IPFS network to download a folder containing the conted required to render the scene
+2) The user agent communicates with known peers or a content server that keeps a properly authorized copy of the files required to render and interact with the scene.
+
+In other words, the user agent follows this algorithm to determine what's the current parcel manifest for a scene:
+
+![](dsp-0002/source-of-truth.png)
+
+### Verification of a Parcel Manifest's Authenticity
+
+If a random peer or website has provided a parcel manifest file, along with a signature (which follows the structure of an IPNS signed message with an secp256k1 signature), this is the algorithm that should be followed in order to verify its authenticity:
+
+![](dsp-0002/verify-signature.png)
+
+#### Decentraland's IPFS Content Server HTTP API
+
+Most of the Decentraland implementations currently use a federation of content servers in order to facilitate faster access to scene files than another distributed method.
+
+See https://github.com/decentraland/content-service for more information.
+
+### Algorithm to load surrounding parcels
+
+The current implementations of Decentraland use the following algorithm to keep a list of surrounding parcels. The global state of this algorithm contains these variables:
+
+* `position`: The player's position
+* `LoadedScenes`: The set of scenes that have been loaded or started loading
+* `WorldMapper`: A service that maps LAND coordinates to the most recent and valid Scene Manifest received.
```

Comment:
Add: World mapper also validates the scene manifest and avoid overlaps with other scenes